v3/examples/mqtt-outbound/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	spinhttp "github.com/spinframework/spin-go-sdk/v3/http"
+	"github.com/spinframework/spin-go-sdk/v3/mqtt"
+)
+
+func main() {}
+
+func init() {
+	spinhttp.Handle(func(w http.ResponseWriter, r *http.Request) {
+		var missingEnvs []string
+		addr := validateEnv("MQTT_ADDRESS", &missingEnvs)
```

Comment:
```suggestion
		addr := os.Getenv("MQTT_ADDRESS")
```

For an example, I would keep it simple and remove the logic around missing envars